final-project/src/tier1/main.py

```diff
@@ -2,22 +2,20 @@
 from aerial_gym.envs import *
 from aerial_gym.utils import get_args, task_registry
 import torch
-from aeriel_robot_cfg_final_project import AerialRobotCfgFinalProjectTier1
-from aeriel_robot_final_project import AerialRobotFinalProjectTier1
 from action_primitives import QuadActionPrimitives
 from deep_q_learning import DeepQLearningAgent, State
 from reward_system import QuadRewardSystem
+from envs import *
 
 class GymInterface:
```

Comment:
This class probably should be in its own file too, since you've moved all other classes to separate files. main.py can just combine all that.

Reply:
This one I guess you don't have to take care of within this PR.

Reply:
Yes, left for next PR.